src/main/java/com/saucedemo/pages/DriverFactory.java

```diff
@@ -15,7 +16,11 @@ public static WebDriver getDriver() {
         switch (browser) {
             case "chrome":
                 WebDriverManager.getInstance(CHROME).setup();
-                driver = new ChromeDriver();
+                ChromeOptions options = new ChromeOptions();
```

Comment:
Two things. 
-> When we work on configuration, remember to take these out in config file so that if you want to overwride these in local or ci, you can easily do that. For example, when testing from local, you would want "headless" mode to be false, else you will not see the browser. For now, it is okay since we have not yet worked on configuration story fully.
-> Play with chrome options and see if removing any of these still makes CI pass or if all of these are absolutely necessary for CI.

Reply:
Noted. We still need to discuss my solution with config factory. I will make this update on that PR once we discuss the base solution.